models/users.model.js

```diff
@@ -0,0 +1,12 @@
+const db = require("../db/connection.js");
+
+exports.selectAllUsernames = () => {
+  return db
+    .query(`SELECT username FROM users`)
+    .then(({ rows }) => {
+      return rows;
+    })
+    .catch((err) => {
+      console.log(err);
+    });
```

Comment:
You'll want to remove the `catch` here Ben - it will get in the way of the catch in the controller which is where you want to pass an error to `next` - there's not really an error I could foresee without something being up with the database itself - but you never know!